### PR TITLE
✨ Add Search Page

### LINKED ISSUE #7: Search Page
### User Story

As a User,
i would like to have a function in which I can filter by categories and locations or search for terms of my own. These should then be listed with the main information. At the end of the page i would like a navigation
to switch between the main pages.

### Screenshots

Bildschirmfoto 2021-06-13 um 02 30 10

### Tasks

- [x] Headline
- [ ] Input field filer categories
- [ ] Input field filter locations
- [ ] Input field search
- [ ] list Project cards with main information
- [x] Navbar

### PR DESCRIPTION
**Pull request by dhnazarenko**
fixes #7

- The search page has no function yet.

Check it out 👀 :
https://timeonation-search-page-9ozwre.herokuapp.com/storybook/?path=/story/page-search--default

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

File: src/app/pages/Search/Search.module.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  max-width: 500px;
```
Comment (lmachens) [inline src/app/pages/Search/Search.module.css:4]: Do you have this `max-width` on every page? If so, add it to the `App.tsx`.

**Review by lmachens — APPROVED** [review]
gj

File: src/app/pages/Search/Search.module.css (hunk 1 above)
Comment (dhnazarenko) [reply src/app/pages/Search/Search.module.css:4]: Yes I have that everywhere. But in the end it will be replaced by a mobile layout.

Comment (lmachens) [reply src/app/pages/Search/Search.module.css:4]: So in the end you have to remove it from every page instead of App.tsx only ;)

**Pull request merged**